test/types/readable.test-d.ts

```diff
@@ -1,5 +1,5 @@
 import { expectAssignable } from 'tsd'
-import BodyReadable from '../../types/readable'
+import BodyReadable = require('../../types/readable')
```

Comment:
is it possible for us to test both ways? I want to ensure we are not breaking `allowSyntheticDefaultImports=true` users

Reply:
It is! There is no impact for configurations with the setting enabled.